pkg/config/app/kuma-cp/kuma-cp.defaults.yaml

```diff
@@ -586,6 +586,8 @@ dpServer:
 
 # Intercommunication CP configuration
 interCp:
+  # Enable InterCp component. Default: true.
+  enabled: true
```

Comment:
I don't think it makes sense to this be configurable to the user

Reply:
I'll change it to be a wired in like this - https://github.com/kumahq/kuma/pull/6583#discussion_r1175501080. Or is there another proven method to "private" config options?